src/VisualStudio/NuGet.Packaging.VisualStudio.15/Templates/Projects/Nuget/Package/Package.nuproj

```diff
@@ -14,6 +14,7 @@
 	</ItemGroup>
 	<PropertyGroup Label="Configuration">
 		<PackageId>$safeprojectname$</PackageId>
+		<PackageVersion>1.0.0</PackageVersion>
```

Comment:
Well, `<Configuration Condition="'$(Configuration)' == ''">Debug</Configuration` and friends are all conditioned like that ;)

Without the condition, you wouldn't be able to override the value using environment variables. It would work if you override via `msbuild /p:PackageVersion=...` though, but that would seem inconsistent to me...